### PR TITLE
Update "software" method in Identity Management

### PR DESCRIPTION
**Pull request by phallobst**
See #4410

### CODE REVIEW

File: _data/identity.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ websites:
       - sms
       - phone
       - email
-      - software
+      - totp
       - hardware
```
Comment (Carlgo11) [inline _data/identity.yml]:
```suggestion
      - u2f
```

File: _data/identity.yml [hunk 2]
```diff
@@ -35,14 +35,15 @@ websites:
       - sms
       - phone
       - email
-      - software
+      - totp
+      - hardware
```
Comment (Carlgo11) [inline _data/identity.yml]:
```suggestion
      - u2f
```

File: _data/identity.yml [hunk 3]
```diff
@@ -35,14 +35,15 @@ websites:
       - sms
       - phone
       - email
-      - software
+      - totp
```
Comment (Carlgo11) [inline _data/identity.yml]: I can't find anything about totp support other than that their proprietary app "CENTRIFY MOBILE APP" does MFA using TOTP codes and push notifications.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/identity.yml (hunk 2 above)
Comment (phallobst) [reply _data/identity.yml]: see below

File: _data/identity.yml (hunk 3 above)
Comment (phallobst) [reply _data/identity.yml]: see below

**Comment by phallobst** [comment]
I changed the Centrify doc and updated the tfa section accordingly.

File: _data/identity.yml (hunk 3 above)
Comment (Carlgo11) [reply _data/identity.yml]:
```suggestion
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/identity.yml (hunk 3 above)
Comment (phallobst) [reply _data/identity.yml]: I'm not sure. The doc says:
> OATH Tokens such as managed by Google Authenticator...

This means TOTP, doesn't it?

**Review by Carlgo11 — APPROVED** [review]

**Review by Jawshy — APPROVED** [review]
All sites with documentation ✅

**Comment by phallobst** [comment]
Okay, what happened here? I just merged _master_ to get the "workflow" check running again, and now there are changes all over the place... I must have messed up something.

Shall I close this PR and open a new one with just the "identity.yml" file?

**Pull request closed without merging**